vector/src/main/res/values-fr/strings_tchap.xml

```diff
@@ -52,8 +52,8 @@
     <string name="tchap_scan_media_error_file_is_infected">Le fichier est infecté, impossible de le télécharger.</string>
 
     <!-- tchap room settings -->
-    <string name="tchap_room_settings_remove_from_directory">Retirer ce salon de la liste des forums</string>
-    <string name="tchap_room_settings_remove_from_directory_prompt_msg"><b>Cette action est irréversible.</b>\nVoulez-vous vraiment retirer ce salon de la liste des forums ?</string>
+    <string name="tchap_room_settings_remove_from_directory">Convertir en salon privé (irréversible)</string>
+    <string name="tchap_room_settings_remove_from_directory_prompt_msg"><b>Cette action est irréversible.</b>\nVoulez vous vraiment convertir ce forum en salon privé ?</string>
```

Comment:
I think we can also rename the keys that are not anymore related to removing from directory

Reply:
> I think we can also rename the keys that are not anymore related to removing from directory

@Florian14 @Claire1817 : I suggest to replace "tchap_room_settings_remove_from_directory" with "tchap_room_settings_convert_to_private" in both keys name. Wdyt?